packages/swing-store/src/snapStore.js

```diff
@@ -230,23 +234,27 @@ export function makeSnapStore(
     toDelete.add(hash);
   }
 
-  function commitDeletes(ignoreErrors = false) {
+  async function commitDeletes(ignoreErrors = false) {
```

Comment:
It was needed in an earlier version to allow executing JS while the delete I/Os were taking place. This version is strictly superiors though since it parallelizes the deletes.

Reply:
And you're correct, at this point it became a drive by change.